src/mscorlib/src/System/Runtime/CompilerServices/ConditionalWeakTable.cs

```diff
@@ -150,6 +150,41 @@ public void Add(TKey key, TValue value)
         }
 
         //--------------------------------------------------------------------------------------------
+        // key: key to add or update. May not be null.
+        // value: value to associate with key.
+        //
+        // If the key is already entered into the dictionary, this method will update the value associated with key.
+        //--------------------------------------------------------------------------------------------
+        [System.Security.SecurityCritical]
```

Comment:
We do not care about the transparency annotations in corelib anymore with the mirror turned off. They can be deleted.